src/moduler/aktivitet/visning/hjelpekomponenter/aktivitetinformasjon-visning.js

```diff
@@ -29,7 +35,13 @@ function visningsIngress(type) {
 }
 
 function AktivitetinformasjonVisning({ valgtAktivitet, tillatEndring, laster, underOppfolging }) {
-    const { tittel, type, arenaAktivitet } = valgtAktivitet;
+    const { id, tittel, type, arenaAktivitet, avtalt, erReferatPublisert } = valgtAktivitet;
+
+    const erVeileder = useSelector((state) => selectErVeileder(state));
```

Comment:
Her kan det også skrives på denne måten:
```suggestion
    const erVeileder = useSelector(selectErVeileder);
```

Reply:
Skreiv det slik først, men ble usikker på format siden jeg så første pattern andre steder.